DependencyInjection/DoctrineExtension.php

```diff
@@ -375,6 +378,9 @@ protected function ormLoad(array $config, ContainerBuilder $container)
                 $def->addTag('doctrine.event_subscriber');
             }
         }
+
+        $container->registerForAutoconfiguration(EntityRepository::class)
```

Comment:
Shouldn’t the tag only be applied to the new ServiceEntityRepository?

Reply:
doing so wuld allow removing the config option altogether, which would be nice, isn't it?

Reply:
Interesting... to remove the option entirely, it would mean that, when an entity has a custom repository, we would sometimes instantiate that class (like current behavior) and sometimes fetch it from the container. Effectively, if their repo extends the new ServiceEntityRepository, then we fetch from container (or error if not exists). Otherwise, we instantiate.

I like this because it makes migrating to the new system even easier and removes this option. But is it too magic? The choice of your base class will determine how your repo is fetched/created.

Reply:
That's what autoconfig is already, adding behavior based on the base classes :)